maven-slf4j-provider/src/main/java/org/slf4j/impl/MavenSimpleLogger.java

```diff
@@ -71,8 +78,13 @@ private void printStackTrace(Throwable t, PrintStream stream, String prefix) {
             stream.print(prefix);
             stream.print("    ");
             stream.print(builder().strong("at"));
-            stream.print(" " + e.getClassName() + "." + e.getMethodName());
-            stream.print(builder().a(" (").strong(getLocation(e)).a(")"));
+            stream.print(" ");
+            stream.print(e.getClassName());
+            stream.print(".");
+            stream.print(e.getMethodName());
+            stream.print(" (");
+            stream.print(builder().strong(getLocation(e)));
+            stream.print(")");
```

Comment:
Are you sure this is interesting in terms of performances ? The benefit to avoid multiple buffer allocations may be balanced by the synchronisation / flushes on the underlying character/byte streams...

Reply:
Yes that is right, I noticed this while doing different tests but forgot to push the changes. Pushing the commit now.